### PR TITLE
Resolve outstanding dictionary issues within the SCI repo

### LINKED ISSUE #5: Term: Action
@Henry-WattTime talked about an 'Action' being what the specification should measure, rather than an overall carbon footprint for an application. This makes sense, as I guess there are many different execution paths that a piece of software could take.

Do we have a definition for what an 'Action' is? Something we could put in https://github.com/Green-Software-Foundation/dictionary ?

I guess you may need to define what that is - it sounds to me to be above the level of an operation, but at a level where you could logically identify different use cases for the software.

I'm thinking that having something like an action allows us to identify and quantify 'C' or the total amount or carbon that software is emitting. Would you maybe build up 'C' by identifying the emissions for each action? That could be useful in incentivising optimising particular parts of an application, the most frequently used.

### PR DESCRIPTION
**Pull request by Sealjay**
Adds to Green-Software-Foundation/software_carbon_intensity#32, resolves #5  and resolves #4 - @atg-abhishek

### CODE REVIEW

**Comment by atg-abhishek** [comment]
Yes this gets merged on the call - did it not come up on the agenda in this week's call? I had it written in on the agenda but maybe there wasn't enough time and we didn't get to it? cc @seanmcilroy29 @Henry-WattTime

**Comment by Sealjay** [comment]
Yeah I think it's related to #64 - there's so much going on, it's hard to get to everything on the calls :)

File: Dictionary/Dictionary.md [hunk 1]
```diff
@@ -73,14 +73,34 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 <table>
   <caption>Definitions</caption>
   <tbody>
+        <tr>
+	<td><strong>Ability to Act</strong></td>
```
Comment (SaraEmilyBergman) [inline Dictionary/Dictionary.md:77]: Nitpicking here, but don't we want "Ability to Act" below "Agent" and "Action" as both of those are used in the definition?

Comment (Sealjay) [reply Dictionary/Dictionary.md:77]: Ah we do, yes! When we merge #24, which changes the table format to .MD with @atg-abhishek @TaylorPrewitt, I'll reorder @SaraEmilyBergman  - good catch

**Comment by atg-abhishek** [comment]
We want to keep it in alphabetical order since it is a dictionary and maybe add in internal references to existing terms.

**Comment by Sealjay** [comment]
@atg-abhishek agreed - definitely in alphabetical order, was just waiting for the markdown changes to be merged before I fixed, otherwise it would conflict further

**Comment by Sealjay** [comment]
@atg-abhishek - fixed, and ready to merge. mildly painful!

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
Thanks for working on this @Sealjay - I noticed that the `md` table hasn't rendered correctly, could you take a look at that? Otherwise, this is good to go!

**Review by Sealjay — COMMENTED** [review]
The MD should now render correctly

**Comment by atg-abhishek** [comment]
@Sealjay can you resolve the conflicts and we can move to merge this? Thanks!

**Comment by Sealjay** [comment]
> @Sealjay can you resolve the conflicts and we can move to merge this? Thanks!

should be done now @atg-abhishek and @seanmcilroy29 - had to merge in the two other PRs :)

**Comment by Sealjay** [comment]
hey @atg-abhishek and @seanmcilroy29 - just bumping this one, thanks :)

**Review by atg-abhishek — APPROVED** [review]
LGTM!

**Comment by Sealjay** [comment]
Awesome thanks- is someone with permissions able to merge? @atg-abhishek @seanmcilroy29

**Comment by atg-abhishek** [comment]
I've added it to the agenda for the meeting for review before it can be merged in as per the WoW of the WG :)

**Comment by Sealjay** [comment]
ok, sure - I thought we discussed this in August and September. This is pulling in upstream changes only, no new content.
https://github.com/Green-Software-Foundation/standards_wg/blob/4542e07030082974781aa268df7e9113dd120e90/Agenda_Minutes/2021.09.30_Agenda.md

**Pull request merged**